jbmc/src/Makefile

```diff
@@ -1,4 +1,5 @@
 DIRS = janalyzer jbmc jdiff java_bytecode miniz
+ROOT = ../
```

Comment:
Is this variable necessary? You could obtain the path pointed by it via `$(CPROVER_DIR)`?

Reply:
CPROVER_DIR gives the path to the project root, since the generated files are for JBMC, it kind does not make sense for the distribution folder to be in the root of the repository (it should be in $(CPROVER_DIR)/jbmc

Reply:
That's what I meant, you could use `$(CPROVER_DIR)/jbmc` wherever you use `ROOT`. You can ignore this, it's a small thing.